test/test_von_mises.cpp

```diff
@@ -330,11 +331,29 @@ void test_spots(RealType)
 
   // Error tests:
   check_out_of_range<boost::math::von_mises_distribution<RealType> >(0, 1); // (All) valid constructor parameter values.
-    
+
   BOOST_MATH_CHECK_THROW(quantile(von_mises_distribution<RealType>(0, 1), -1), std::domain_error);
   BOOST_MATH_CHECK_THROW(quantile(von_mises_distribution<RealType>(0, 1), 2), std::domain_error);
 } // template <class RealType>void test_spots(RealType)
 
+template <typename RealType>
+void test_symmetry(RealType)
```

Comment:
This doesn't need an argument does it?

Reply:
I just used the same style that also `test_spots` uses.